packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -138,23 +143,12 @@ const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
               wrapperClassName={iconWrapperClassName}
             />
           )}
-          <Tooltip
-            content={shouldShowTooltip ? finalTooltipContent : null}
-            position={tooltipPosition}
-            showDelay={tooltipShowDelay}
-            {...tooltipProps}
-          >
-            <div ref={titleRef} className={styles.title}>
-              {title}
-            </div>
-            {/* Tooltip should be on a whole MenuItem, but it's a breaking change (tooltip adds span) - should be fixed in the next major and then this div be removed */}
-            <div className={styles.hiddenTitle} aria-hidden tabIndex={-1}>
```

Comment:
the tests did queryAll instead of query or getAll instead of get, as there were 2 texts
if they're passing then leave it, but worth migrating it some day as it would probably be unclear why there's query all and not just query

Reply:
Oh got it, changed them anyway to make sure it will fail otherwise